orb/core/model.py

```diff
@@ -204,6 +244,7 @@ def __init__(self, *info, **context):
 
         # pop off additional keywords
         loader = context.pop('loadEvent', None)
+        delayed = context.pop('delay', False)
```

Comment:
added the ability to delay loading of a model, so doing `a = User(1)` won't immediately query for `id==1`, instead it will wait until it is needing something from the backend to query.  Used in the `pyramid_orb` traversal method to optimize querying down the path